lib/compat/wordpress-5.9/translate-settings-using-i18n-schema.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Function to translate json files.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Translates the provided settings value using its i18n schema.
+ *
+ * @param string|string[]|array[]|object $i18n_schema I18n schema for the setting.
+ * @param string|string[]|array[]        $settings    Value for the settings.
+ * @param string                         $textdomain  Textdomain to use with translations.
+ *
+ * @return string|string[]|array[] Translated settings.
+ */
+function gutenberg_translate_settings_using_i18n_schema( $i18n_schema, $settings, $textdomain ) {
```

Comment:
These functions are already part of the core. Are they part of 5.8 if yes we can just remove the Gutenberg versions and use core.

Reply:
As far as I saw, they were introduced in August, post-5.8 release.